### ISSUE TITLE
Don't make "Input Rules" a heading

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#595]
in the metadata of the rule, Rule Type and Accessibility Requirements are both simply items in a list, but Input Rules, which looks exactly the same is marked up as an `h3`. There is no real reason this should be an h3.

**Comment by jkodu** [comment]
PR in place, closing - https://github.com/act-rules/act-rules.github.io/pull/635
